research-software-directory/RSD-as-a-service
# Update documentation

# Update documentation to latest ui changes

Closes #1136

Changes proposed in this pull request:
* Make new gif animations after ui changes
* Update add software documentation content
* Improve links to documentation pages
* Fix minor type errors found in the ui   

How to test:
* make start to build app and generate data
* check help links on edit software pages (i icons next to section titles)
* check user documentation pages, mainly add/edit software and project pages

PR Checklist:
